portality/tasks/journal_in_out_doaj.py

```diff
@@ -34,8 +36,8 @@ def run(self):
         job = self.background_job
         params = job.params
 
-        journal_ids = params.get("set_in_doaj__journal_ids")
-        in_doaj = params.get("set_in_doaj__in_doaj")
+        journal_ids = self.get_param(params, "journal_ids")
+        in_doaj = self.get_param(params, "in_doaj")
 
         if journal_ids is None or len(journal_ids) == 0 or in_doaj is None:
```

Comment:
After giving this a bit of thought and trying it out on one class, I've decided not to do it.  The checks on prepare and the checks on run are not necessarily the same, and the kwargs to prepare and the params for run are not necessarily the same or consistently named.  After fiddling with it the _job_parameter_check seemed to be a bit more complex than was necessary and also made it less clear what prepare and run were doing in my opinion.